fetch.bs

```diff
@@ -1542,6 +1549,11 @@ whose <a for=request>destination</a> is "<code>audio</code>", "<code>audioworkle
 "<code>script</code>", "<code>style</code>", "<code>track</code>", "<code>video</code>",
 "<code>xslt</code>", or the empty string.
 
+<p>A <dfn export>stale permitting subresource request</dfn> is a <a for=/>request</a>
+whose <a for=request>destination</a> is "<code>font</code>", "<code>image</code>",
+"<code>script</code>", "<code>style</code>", "<code>xslt</code>", or
+<a for=request>initiator</a> is "<code>prefetch</code>".
```

Comment:
Allowing stale request for prefetch feels slightly odd to me because responses for prefetch won't be used downstream (per definition).  This one can just trigger a revalidation therefore the next request (the real one that wants to use the response) could use a fresh resource?

Reply:
Yes I agree. Removed.